### PR TITLE
feat/donate dialog when downloading grass

### PR DESCRIPTION
**Pull request by nobeeakon**
# what
 - show donate dialog when clicking the download links. 
 -  Added it in Linux, windows and Mac download pages
 - Used text and images suggested in #316 
 - Minor refactor to the headers in in Mac, Windows Linux and Docker download pages, replaced `<a name="...">` with no content with `<span id="...">` 

imagen

# issue
#316

### CODE REVIEW

**Comment by veroandreo** [comment]
I just tested and seems to work as expected for linux, window and mac, but the dialog does not appear after clicking in the links of the docker page. Is that ok?

**Comment by nobeeakon** [comment]
yeah, didn't include the Dialog in the Docker page as it is linking to another site (mundialis) and was a bit confused about that... anyways, I can add it there too

**Comment by nobeeakon** [comment]
done, added the Dialog in the Docker download page

File: themes/grass/layouts/shortcodes/donateDialog.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            While downloading your <em>free</em> and <em>open source</em> copy of <strong>GRASS GIS</strong>, if you or your 
+            institution enjoy the software 
+            please consider making a  
+            <a href="https://opencollective.com/osgeo/projects/grass#section-contribute">
+              <strong>
+                donation to support the project</strong></a>.
+             May the FOSS be with you!
+          </p>
+          <p>
+            <i>GRASS Team</i>
```
Comment (veroandreo) [inline themes/grass/layouts/shortcodes/donateDialog.html:35]:
```suggestion
            <i>GRASS GIS Team</i>
```

**Pull request merged**